bnd.bnd

```diff
@@ -0,0 +1,28 @@
+Export-Package:\
```

Comment:
Every versioned package is automatically exported

Reply:
Thanks for the review.  This was a direct copy of how the old maven-bundle-plugin instructions were doing it.  And yes, you are correct, but only one of those exported packages contained a package-info.java file so a little more work is needed here.

I have cleaned this up by adding a package-info.java file to the other two packages and removed this instruction.